src/app/@ansyn/imagery/imagery/manager/imagery.component.manager.ts

```diff
@@ -24,22 +24,29 @@ export class ImageryComponentManager {
 	private _subscriptions = [];
 	public centerChanged: EventEmitter<GeoJSON.Point> = new EventEmitter<GeoJSON.Point>();
 	public positionChanged: EventEmitter<CaseMapPosition> = new EventEmitter<CaseMapPosition>();
-	public mapComponentInitilaized: EventEmitter<any> = new EventEmitter<any>();
 	public singleClick: EventEmitter<any> = new EventEmitter<any>();
 	public contextMenu: EventEmitter<any> = new EventEmitter<any>();
 	public mapInstanceChanged: EventEmitter<MapInstanceChanged>;
-
 	public activeMapName: string;
-	private _plugins: IMapPlugin[] = [];
 	private _visualizers: IMapVisualizer[] = [];
 
+	// protected properties
```

Comment:
Did you mean public?

Reply:
No, only getters, no setter..
I'll remove this comment to avoid confusion